packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -185,11 +184,12 @@ export const DatePicker: React.FC<DatePickerProps> = ({
   const inputRef = useRef<HTMLInputElement>(null)
 
   const { hasFocus, onLocalBlur, onLocalFocus } = useFocus()
-  const {
-    setFalse: close,
-    value: isOpen,
-    toggle: toggleIsOpen,
-  } = useBoolean(initialIsOpen)
+  const [isOpen, setIsOpen] = useState(initialIsOpen)
```

Comment:
What about the `useOpenClose` hook: https://github.com/defencedigital/mod-uk-design-system/blob/master/packages/react-component-library/src/hooks/useOpenClose.ts

Reply:
That one does some other stuff as well – IIRC it got a bit complex with focus-trap and also the reducer and it was simplified. `useOpenClose` also has a `useEffect` which doesn't make too much sense as the DatePicker manages its own open/close state after the initial state is set.

In any case, I need to work out why the TabSet e2e test is suddenly failing (unrelated I guess...)